### DISCUSSION TITLE
case insensitive when searching a value

### DISCUSSION THREAD

**Discussion by giulianoc** [open-source-parsers/jsoncpp#1346] [General]
Hi,
is it possible to have a case insensitive search when a field is searched?
For example, if I have the statements:

Json::Value sourceToBeEncodedRoot;
string myString = myJsonRoot.get("myField", "").asString();

Is it possible by Json::Value to search "myField" in a case insensitive way?

Best regards

**Comment by BillyDonahue** [comment]
> Is it possible by Json::Value to search "myField" in a case insensitive way?

There's no way to do this without changing `Json::Value`.

You could hack `Json::Value::CZString`'s `operator<` or give a custom comparator to the `ObjectValues` map to achieve this in your own fork I guess.
